backend/location/utility.js

```diff
@@ -0,0 +1,43 @@
+const Property = require('./property');
+
+/* Utility object stores the information about a utility location.
+ * @param name String name of the location
+ * @param boardPreset JSON with data loaded from the stored board
+ */
+class Utility extends Property {
+
+    // set up initial state
+    constructor(name, boardPreset) {
+        super(name, boardPreset);
+
+        // TODO decide if needs other features
+
+    }
+
+    /* Sees how much rent costs if this is landed on.
+     * @param numOwned int of number of other utilities owned
+     * 
+     * @return int value
+     */
+    getRent(numOwned) {
+        // let dice = amount shown on dice
```

Comment:
this.rent should have the multiplier stored in it so you don't need all of these if statements